### PR TITLE
PLAT-130867: SwatchButton: Fixed invalid prop `children` of type `object` supplied, expected `string` error

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Removed `MarqueeDecorator` from `SwatchButtonDecorator `.

### Links
[//]: # (Related issues, references)
PLAT-130867

### Comments

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 ### Fixed
 
 - `agate/Checkbox` style to match latest design for Silicon skin
+- `agate/ColorPicker/SwatchButton` to not show console error in sampler
```
Comment (MikyungKim) [inline CHANGELOG.md:10]: Please remove the changelog since this component is private.

File: ColorPicker/SwatchButton.js [hunk 2]
```diff
@@ -112,7 +115,11 @@ const SwatchButtonBase = kind({
  * @public
  */
 const SwatchButtonDecorator = compose(
-	ButtonDecorator,
+	Pure,
+	UiButtonDecorator,
+	IconButtonDecorator,
+	TooltipDecorator({tooltipDestinationProp: 'decoration'}),
```
Comment (MikyungKim) [inline ColorPicker/SwatchButton.js:121]: I think we don't need to use these decorators for this component. Let's remove them and add `iconOnly` prop to `ButtonBase`.

Comment (MikyungKim) [reply ColorPicker/SwatchButton.js:121]: at line 101.

File: ColorPicker/SwatchButton.js [hunk 3]
```diff
@@ -112,7 +115,11 @@ const SwatchButtonBase = kind({
  * @public
```
Comment (MikyungKim) [inline ColorPicker/SwatchButton.js:115]: Please update jsdoc above.

File: Button/Button.js [hunk 4]
```diff
@@ -369,5 +369,6 @@ export default Button;
 export {
 	Button,
 	ButtonBase,
-	ButtonDecorator
+	ButtonDecorator,
+	IconButtonDecorator
```
Comment (MikyungKim) [inline Button/Button.js:373]: Please don't make a change here.

**Comment by stanca-pop-lgp** [comment]
@MikyungKim when you said that I should fix JS docs, did you refer to changing the component to @private as well? Because I tried and got a Travis build error:
image

**Comment by MikyungKim** [comment]
> @MikyungKim when you said that I should fix JS docs, did you refer to changing the component to @Private as well? Because I tried and got a Travis build error:
> image

No, the current change looks fine.

**Review by MikyungKim — APPROVED** [review]
LGTM!

**Pull request merged**